react/features/base/conference/middleware.js

```diff
@@ -116,16 +116,12 @@ StateListenerRegistry.register(
             maxReceiverVideoQuality,
             preferredReceiverVideoQuality
         } = currentState;
-        const changedPreferredVideoQuality = preferredReceiverVideoQuality
-            !== previousState.preferredReceiverVideoQuality;
-        const changedMaxVideoQuality = maxReceiverVideoQuality
-            !== previousState.maxReceiverVideoQuality;
+        const changedPreferredVideoQuality
+            = preferredReceiverVideoQuality !== previousState.preferredReceiverVideoQuality;
+        const changedMaxVideoQuality = maxReceiverVideoQuality !== previousState.maxReceiverVideoQuality;
 
         if (changedPreferredVideoQuality || changedMaxVideoQuality) {
-            _setReceiverVideoConstraint(
-                conference,
-                preferredReceiverVideoQuality,
-                maxReceiverVideoQuality);
+            _setReceiverVideoConstraint(conference, preferredReceiverVideoQuality, maxReceiverVideoQuality);
```

Comment:
Please, in the future, start doing these formatting changes in a separate commit or PR, as it can get confusing about what changed and why. As it is right now, the PR is about optimizing resizes and I don't get how this is part of optimizing resizes.

Reply:
I'm sorry!